### DISCUSSION TITLE
使用max build 后可以看到源码

### DISCUSSION THREAD

**Discussion by xu756** [umijs/umi#12752] [Q&A]
```
pnpm build（生产环境）后 为什么可以在前台看到源码
```
<img width="728" alt="image" src="https://github.com/user-attachments/assets/9a38d581-dd9f-4b2b-85ca-5d4bac6c1df3">

**Comment by ghost** [comment]
我猜配置加上了devtool 所以產生了sourcemap 
[devtool](https://umijs.org/docs/api/config#devtool)
